### DISCUSSION TITLE
AFN News

### DISCUSSION THREAD

**Discussion by baliebox4916** [iptv-org/community#823] [Q&A]
### Channel ID

AFNNews.us

### Notes

a Request for a Package Feat The Armed Forces Networks TV Stations That Includes

AFN Prime
AFN Spectrum
AFN Pulse
AFN News
AFN Sports
AFN Sports 2
AFN Movies

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19773
